ibm/data_source_ibm_is_security_group_rules_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMIsSecurityGroupRulesDataSourceBasic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMIsSecurityGroupRulesDataSourceConfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rules.is_security_group_rules", "id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rules.is_security_group_rules", "security_group_id"),
```

Comment:
no need to check for ids. Check for the contents inside rules list

Reply:
done.